examples/amp-consent/cmp-vendors.amp.html

```diff
@@ -185,6 +211,30 @@ <h3>Image that is NOT blocked by consent</h3>
         </amp-consent>
         <!-- End didomi example -->
 
+      <!-- iubenda example -->
+      <amp-consent id="iubenda" layout="nodisplay" type="iubenda">
+        <!--
+          If you want to request consent only to EU users then replace "consentRequired": true with "promptIfUnknownForGeoGroup": "eu" -> allows to ask consent only to EU users.
+        -->
```

Comment:
nit; remove these comments since we have them in your documentation.

Reply:
Done.